src/sentry/api/event_search.py

```diff
@@ -215,9 +215,9 @@ class SearchVisitor(NodeVisitor):
         'device.battery_level', 'device.charging', 'device.online',
         'device.simulator', 'error.handled', 'issue.id', 'stack.colno',
         'stack.in_app', 'stack.lineno', 'stack.stack_level',
-        # OrganizationEvents aggregations
-        'event_count', 'user_count',
-
+        'duration',
```

Comment:
Is this going to break regular event/issue search if someone is using these field names?

Reply:
The removed fields were only really available on events-v2, which no customers have access to.

Reply:
This code is not just used in events-v2 though, it's used for event search generally. Try typing `duration:something` into issue or event search and you will see what I mean

Reply:
I doubt anyone knows about these fields since they were never documented

Reply:
Unfortunately people do use `duration` as a tag though. And 500ing on a query with this term in it doesn't seem great

Reply:
I can remove duration for now. Given that some customers are using this as a tag would we want to expose the real duration as `event.duration`?